docker/mosaic.json

```diff
@@ -4,8 +4,8 @@
       "gatewayLibAddress": "0x353AEba14b902e67cDdA99595f74d910e3FD5e97",
       "messageBusAddress": "0xB69f2Aa06793028CC8E604122218b1D2E23eae4b",
       "merklePatriciaLibAddress": "0xf8238D9850F2a47f1411Cb7F5F5F3279A72b7f7a",
```

Comment:
also please run mosaic-verify for all these mosaic-configs

Reply:
looks like mosaic-verify does not take `config-path` as param and always defaults to verifying config for baseToken.

Reply:
We might have to start taking as CLI input

Reply:
We have a ticket for this: https://github.com/mosaicdao/mosaic-chains/issues/94
We can verify with above ticket.